lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mmap_postings.rs

```diff
@@ -92,11 +92,11 @@ impl MmapPostings {
     /// _alignment: &'a [u8], // 0-3 extra bytes to align the data
     /// remainder_postings: &'a [PointOffsetType],
     /// ```
-    fn get_reader(
-        &self,
-        header: &PostingListHeader,
-        hw_counter: &HardwareCounterCell,
-    ) -> Option<ChunkReader<'_>> {
+    fn get_reader<'a>(
+        &'a self,
+        header: &'a PostingListHeader,
```

Comment:
`header` doesn't need a lifetime here, since it is not part of the returned value.